datm.streams.xml

```diff
@@ -9,78 +14,78 @@
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_199501010130-199512312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_199601010130-199612312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_199701010130-199712312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_199801010130-199812312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_199901010130-199912312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200001010130-200012312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200101010130-200112312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200201010130-200212312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200301010130-200312312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200401010130-200412312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200501010130-200512312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200601010130-200612312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200701010130-200712312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200801010130-200812312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_200901010130-200912312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201001010130-201012312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201101010130-201112312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201201010130-201212312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201301010130-201312312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201401010130-201412312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201501010130-201512312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201601010130-201612312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201701010130-201712312230.nc</file>
-      <file>./INPUT/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_201801010130-201812312230.nc</file>
+      <file>./INPUT/atmos/3hr/prsn/gr/v20190429/prsn_input4MIPs_atmosphericState_OMIP_MRI-JRA55-do-1-4-0_gr_195801010130-195812312230.nc</file>
```

Comment:
@ezhilsabareesh8 - is it intentional that `atmos/3hr/prsn/gr/v20190429/` got moved into this file ?

Reply:
Yes it is, Dougie restructured the input directories in `vk83` to avoid many input paths in `config.yaml`